client/js/socket-events/search.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const $ = require("jquery");
+const socket = require("../socket");
+const render = require("../render");
+const messages = $("#search-query .results .messages");
+const form = $("#search-query form");
+const select = $("#search-query form select");
+
+socket.on("search-networks-channels", function(data) {
+	var groups = {};
+	select.empty();
+	select.append($("<option></option>").attr("value", "*/*").append("** All networks and channels **"));
+	data.forEach(function(entry) {
+		if (!groups[entry.network]) {
+			groups[entry.network] = [];
+		}
+		if (entry.network !== entry.chan) {
+			groups[entry.network].push(entry.chan);
+		}
+	});
+	Object.keys(groups).forEach(function(network) {
+		var group = $("<optgroup></optgroup>").attr("label", network);
```

Comment:
let

Reply:
Eslint advise to use const in that case, what's your opinion?

Reply:
You should go with `const` then :)